lib/ansible/executor/task_queue_manager.py

```diff
@@ -290,29 +288,27 @@ def send_callback(self, method_name, *args, **kwargs):
             ]
             for method in methods:
                 if method is not None:
-                    try:
-                        # temporary hack, required due to a change in the callback API, so
-                        # we don't break backwards compatibility with callbacks which were
-                        # designed to use the original API
-                        # FIXME: target for removal and revert to the original code here
-                        #        after a year (2017-01-14)
-                        if method_name == 'v2_playbook_on_start':
-                            import inspect
-                            (f_args, f_varargs, f_keywords, f_defaults) = inspect.getargspec(method)
-                            if 'playbook' in f_args:
-                                method(*args, **kwargs)
-                            else:
-                                method()
-                        else:
+                    # temporary hack, required due to a change in the callback API, so
+                    # we don't break backwards compatibility with callbacks which were
+                    # designed to use the original API
+                    # FIXME: target for removal and revert to the original code here
+                    #        after a year (2017-01-14)
+                    if method_name == 'v2_playbook_on_start':
+                        import inspect
+                        (f_args, f_varargs, f_keywords, f_defaults) = inspect.getargspec(method)
+                        if 'playbook' in f_args:
                             method(*args, **kwargs)
-                    except Exception as e:
-                        import traceback
-                        orig_tb = to_unicode(traceback.format_exc())
-                        try:
-                            v1_method = method.replace('v2_','')
+                        else:
+                            method()
+                    else:
+                        method(*args, **kwargs)
+                else:
```

Comment:
The else clause will occur twice so the callbacks will get called twice.  This is because we're inside of a loop that's iterating through the callback methods for method_name and v2_on_any here.

Perhaps we should check whether either of the methods is None at the top and look for the v1 method there?